jccli/cli/system.py

```diff
@@ -0,0 +1,81 @@
+import json
+import click
+from jccli.jc_api_v1 import JumpcloudApiV1
+
+
+@click.group()
+@click.pass_context
+def system(ctx):
+    """
+    Command set for systems.
+    """
+    pass
+
+
+@system.command("get")
+@click.option('--id', '-i', 'system_id', required=True, type=str)
+@click.pass_context
+def get_system(ctx, system_id):
+    """
+    Detail view of system.
+    """
+    api1 = JumpcloudApiV1(ctx.obj.get('key'))
+    response = api1.get_system(system_id=system_id)
+    serialized_response = json.dumps(response, indent=2)
+    click.echo(f"{serialized_response}")
+
+
+@system.command('list')
+@click.option('--active/--inactive', type=bool, default=None)
+@click.option('--arch', type=str, default=None, help='architecture')
+@click.option('--display-name', 'displayName', type=str, default=None)
+@click.option('--hostname', type=str, default=None)
+@click.option('--os', type=str, default=None)
+@click.option('--remote-ip', 'remoteIp', default=None)
+@click.pass_context
+def list_systems(ctx, **kwargs):
+    """
+    List view of systems. Show systems which match the selected options. Return all systems if no options are selected.
+    """
+    filter = {}
+    for field_name, value in kwargs.items():
+        if value is not None:
+            filter[field_name] = value
+
+    api1 = JumpcloudApiV1(ctx.obj.get('key'))
+    response = api1.search_systems(filter)
+    serialized_response = json.dumps(response, indent=2)
+    click.echo(f"{serialized_response}")
+
+
+@system.command('set')
+@click.option('--id', '-i', 'system_id', required=True, type=str)
+@click.option('--allow-multi-factor-authentication/--disallow-multi-factor-authentication', type=bool, default=None)
+@click.option('--allow-public-key-authentication/--disallow-public-key-authentication', type=bool, default=None)
+@click.option('--allow-ssh-password-authentication/--disallow-ssh-password-authentication', type=bool, default=None)
+@click.option('--allow-ssh-root-login/--disallow-ssh-root-login', type=bool, default=None)
+@click.option('--display-name', type=str, default=None)
+@click.option('--tags', type=str, multiple=True, default=(None,))
```

Comment:
It's not obvious how to set this parameter.  could you please add `help` here with an example?

Reply:
I think this feature (`tags` attribute) is actually [discontinued](https://jumpcloud.com/blog/welcome-to-groups), so I deleted it.